airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/integrations/util/HostPortResolver.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+package io.airbyte.cdk.integrations.util
+
+import java.util.*
```

Comment:
We should avoid this/make sure we have a formatting rule that atomically finds and catches this.  There is no reason to import everything in a package if we don't need it -- it just adds extra memory use.